index.html

```diff
@@ -37,5 +37,6 @@
       <h4>&copy;rue.TH</h4>
     </div>
   </footer>
+  <script type="text/javascript" src="/js/main.js"></script>
```

Comment:
Be careful about the leading `/` in your `src` here. It works on some machines, but doesn't on others. In my case I had to delete it to get the javascript file to load. Leaving it off will fix that across all machines.